src/client/src/rt-components/loadable/Loadable.tsx

```diff
@@ -44,9 +44,9 @@ const Loadable: React.FC<Props> = ({
   minHeight,
 }) => {
   useEffect(() => {
-    onMount && onMount()
+    if (status === ServiceConnectionStatus.CONNECTED && onMount) onMount()
```

Comment:
I think you will have an issue with operator precedence here: `===` has lower precedence than `&&` I think. See https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Operator_Precedence

Reply:
Interesting link, thanks Riko.
In the table === appears before && which means it has highest precedence than && ?
But to make it clearer I can add parenthesis.

Reply:
Prettier removes the additional parenthesis, so it means it's advised not to put them when it's not necessary...

Reply:
doh, I probably have it the wrong way round then. yup, I read it upside down 🤦